bin/sozo/src/main.rs

```diff
@@ -51,5 +54,9 @@ fn cli_main(args: SozoArgs) -> Result<()> {
         .compilers(compilers)
         .build()?;
 
-    commands::run(args.command, &config)
+    trace!(target: LOG_TARGET, "Configuration built successfully.");
```

Comment:
```suggestion
    trace!(target: LOG_TARGET, %manifest_path, "Configuration built successfully.");
```
Always handy to at least have the manifest path if it fails during the command. This will require a clone at line `48`.